docs/changelog.txt

```diff
@@ -42,6 +42,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 - Fixed special characters in `command-prompt` and other non-console in-game outputs on Linux/macOS (in tools using ``df2console``)
 - `dwarfmonitor`, `manipulator`: fixed stress cutoffs
 - `startdwarf`: fixed on 64-bit Linux
+- `die`: Fixed windows crash in the exit handling
```

Comment:
Single backticks in RST produce links, in case you were unaware. This requires an anchor named "die" in another file. I think there is one, but there isn't always for C++-implemented commands, so I'll double-check. Just something to keep in mind.

Reply:
Core.rst:
```rst
.. _die:

die
---
Instantly kills DF without saving.
```
looks good

Reply:
I checked it exists by generating documentations (I have documentations enabled always to verify links which have been broken couple of times for me.)

Reply:
There's also docs/build.sh if you don't want to build docs every time that they change when you're working on other things (particularly scripts).

Reply:
I would just forget it. It is better for me to have everything behind as few commands as possible. If I want to optimize my testing temporary then I prefer to use a specific build target to avoid reduce the incremental build time.